src/mpid/ch4/src/ch4r_rma_target_callbacks.c

```diff
@@ -35,7 +35,7 @@ static int get_acc_iov_target_cmpl_cb(MPIR_Request * rreq);
 static int cswap_target_cmpl_cb(MPIR_Request * rreq);
 static int acc_target_cmpl_cb(MPIR_Request * rreq);
 static int get_acc_target_cmpl_cb(MPIR_Request * rreq);
-static int get_ack_target_cmpl_cb(MPIR_Request * rreq);
+static int get_ack_target_cmpl_cb(MPIR_Request * greq);
```

Comment:
naming here could be confusing given the existence of generalized requests. would suggest just `req` or `get_req`.